examples/pnet-ipfs/index.js

```diff
@@ -1,145 +0,0 @@
-/* eslint no-console: ["off"] */
-'use strict'
-
-const IPFS = require('ipfs')
-const assert = require('assert').strict
-const { generate: writeKey } = require('libp2p/src/pnet')
-const path = require('path')
-const fs = require('fs')
-const privateLibp2pBundle = require('./libp2p-bundle')
-const { mkdirp } = require('./utils')
-
-// Create two separate repo paths so we can run two nodes and check their output
-const repo1 = path.resolve('./tmp', 'repo1', '.ipfs')
-const repo2 = path.resolve('./tmp', 'repo2', '.ipfs')
-mkdirp(repo1)
-mkdirp(repo2)
-
-// Create a buffer and write the swarm key to it
-const swarmKey = Buffer.alloc(95)
-writeKey(swarmKey)
-
-// This key is for the `TASK` mentioned in the writeFileSync calls below
-const otherSwarmKey = Buffer.alloc(95)
-writeKey(otherSwarmKey)
-
-// Add the swarm key to both repos
-const swarmKey1Path = path.resolve(repo1, 'swarm.key')
-const swarmKey2Path = path.resolve(repo2, 'swarm.key')
-fs.writeFileSync(swarmKey1Path, swarmKey)
-// TASK: switch the commented out line below so we're using a different key, to see the nodes fail to connect
```

Comment:
The README mentions the `TASK` comments in the code for trying out the different swarm keys. How this is being done has changed, can we add a `TASK` comment for replacing `const node2 = await privateLibp2pNode(swarmKey)` with `const node2 = await privateLibp2pNode(otherSwarmKey)`. The code is there, there's just no comment saying to do this.

Reply:
good catch